<experimental/optional> is deprecated in Xcode 10.2 and no longer available
in Xcode 11.0; replacement is to use <optional>
See https://trac.macports.org/ticket/61031

--- src/3rdparty/mapbox-gl-native/include/mbgl/util/optional.hpp.orig
+++ src/3rdparty/mapbox-gl-native/include/mbgl/util/optional.hpp
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <experimental/optional>
+#include <optional>
 
 namespace mbgl {
 
